plugins/module_utils/entity.py

```diff
@@ -412,7 +412,7 @@ def _fetch_url(
 
         if resp_json is None:
             self.module.fail_json(
-                msg="Failed to convert API response to json",
+                msg="{0}".format(info.get("msg")),
```

Comment:
Can we have a default message if info doesn't have msg

Reply:
Absolutely, I'll add it in a couple of days.

Reply:
updated @bhati-pradeep